pkg/files/playbook.go

```diff
@@ -82,6 +82,10 @@ func (p *playbooks) GetDefault() (playbook.Inventory, error) {
 }
 
 func (p *playbooks) initFuncMap(t *template.Template) {
+	f := sprig.TxtFuncMap()
+	delete(f, "env")
+	delete(f, "expandenv")
+
 	funcMap := make(template.FuncMap, 0)
 
 	funcMap["sha256sum"] = func(input string) string {
```

Comment:
I think you can remove this function because it's already define in the lib here https://github.com/Masterminds/sprig/blob/4241ae82dd07e5d3391a83c25b79c04450eb22b0/functions.go#L157